homeassistant/components/switch/mochad.py

```diff
@@ -14,6 +14,7 @@
 from homeassistant.const import (CONF_NAME, CONF_DEVICES,
                                  CONF_PLATFORM, CONF_ADDRESS)
 from homeassistant.helpers import config_validation as cv
+from pymochad.exceptions import MochadException 
```

Comment:
Move this import to inside each method where the exception is used. Imports from libraries that are specified in the requirements list need to be done inside each method since the requirements will be installed and made available during setup, ie not before starting home assistant.

Reply:
Moved it as requested. See commit b1f53449